lib/js/components/PopOver/PopOver.vue

```diff
@@ -12,13 +12,25 @@
 	>
 		<div
 			class="popper popOver"
-			:class="{ '-color-neutral': color === POP_OVER_COLORS.NEUTRAL }"
+			:class="{
+				'-color-neutral': color === POP_OVER_COLORS.NEUTRAL,
+				'-small': size === POP_OVER_SIZES.SMALL,
+				'-medium': size === POP_OVER_SIZES.MEDIUM,
+			}"
 		>
 			<img v-if="headerImageUrl" class="popOver__image" :src="headerImageUrl" alt="" />
 			<div class="popOver__content">
 				<div v-if="titleText" class="popOver__title"> {{ titleText }} </div>
 				<div v-if="subtitleText" class="popOver__subtitle"> {{ subtitleText }} </div>
-				<slot :close="close" />
+				<div
+					class="popOver__contentSlot"
+					:class="{
+						'-smallMaxHeight': size === POP_OVER_SIZES.SMALL && maxHeight,
```

Comment:
why don't you use class defined to root element for styling??

Reply:
What do you mean? The root element is `vue-popper`. Where do you suggest I should put these styles?

Reply:
Ok, I wasn't clear. I meant `.popOver` - you have size defined there. IMO, no need to duplicate it in children